clang/lib/CodeGen/CodeGenFunction.cpp

```diff
@@ -2081,7 +2081,26 @@ void CodeGenFunction::EmitBranchOnBoolExpr(
     Weights = createProfileWeights(TrueCount, CurrentCount - TrueCount);
   }
 
-  Builder.CreateCondBr(CondV, TrueBlock, FalseBlock, Weights, Unpredictable);
+  auto *BrInst = Builder.CreateCondBr(CondV, TrueBlock, FalseBlock, Weights,
```

Comment:
nit: https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable
```suggestion
  llvm::Inst *BrInst = Builder.CreateCondBr(CondV, TrueBlock, FalseBlock, Weights,
```